docker/circleci/Dockerfile

```diff
@@ -2,34 +2,44 @@ FROM circleci/golang:1.15-node-browsers
 
 WORKDIR /tmp
 
-# Install musl cross-compiler for ARM64.
 ENV MUSL_VERSION=1.1.24
-ENV MUSL_CROSS_MAKE_VERSION=v0.9.9
+ENV MUSL_CROSS_MAKE_VERSION=0.9.9
 
 RUN sudo apt-get update && \
     sudo apt-get install -y --no-install-recommends \
         patch && \
     sudo rm -rf /var/lib/apt/lists/*
 
-RUN git clone https://github.com/richfelker/musl-cross-make.git && \
-    cd musl-cross-make && \
-    git checkout ${MUSL_CROSS_MAKE_VERSION} && \
-    make MUSL_VER=${MUSL_VERSION} TARGET=aarch64-unknown-linux-musl install && \
-    sudo mv output /usr/local/musl-cross && \
-    cd .. && \
-    rm -rf musl-cross-make
+# Install musl cross-compiler for ARM64.
+# The archive downloaded below was built using these commands:
+#RUN git clone https://github.com/richfelker/musl-cross-make.git && \
+#    cd musl-cross-make && \
+#    git checkout v${MUSL_CROSS_MAKE_VERSION} && \
+#    make MUSL_VER=${MUSL_VERSION} TARGET=aarch64-unknown-linux-musl install && \
+#    mv output musl-cross && \
+#    tar czf /tmp/musl-cross-aarch64-${MUSL_CROSS_MAKE_VERSION}.tar.gz musl && \
+#    cd .. && \
+#    rm -rf musl-cross-make
```

Comment:
Building these archives from source takes > 1 hour. Moving the outputs into S3 reduces build time to O(minutes).